test/cluster/test_connection_metadata.py

```diff
@@ -0,0 +1,79 @@
+# Copyright (C) 2025-present ScyllaDB
+#
+# SPDX-License-Identifier: LicenseRef-ScyllaDB-Source-Available-1.0
+from cassandra.policies import WhiteListRoundRobinPolicy
+from test.cluster.conftest import skip_mode
+from test.pylib.manager_client import ManagerClient
+from test.pylib.random_tables import RandomTables, Column, IntType
+from test.pylib.rest_client import inject_error_one_shot
+from test.pylib.util import wait_for_first_completed
+from test.cluster.util import wait_for_token_ring_and_group0_consistency
+
+import asyncio
+from datetime import datetime, timedelta
+import pytest
+import logging
+import time
+from test.pylib.util import wait_for
+from cassandra.query import SimpleStatement, ConsistencyLevel
+
+logger = logging.getLogger(__name__)
+@skip_mode('release', 'error injections are not supported in release mode')
+@pytest.mark.asyncio
+
+async def test_connection_metadata(request, manager: ManagerClient):
+    # Start with 3 servers
```

Comment:
why not make it use less resources and start with 1 node, add second, remove first? is this testing something particular to 3 nodes?